client/components/Application/Dialogs/FileUpload/FileUpload.store.tsx

```diff
@@ -37,14 +37,14 @@ export function resetState() {
   }
 }
 
-export async function ingestFiles(props: { source: FileList | string }) {
+export async function ingestFiles(props: { source: FileList | string }, t: any) {
```

Comment:
There is no need to pass `t` to function outside of React, it can be imported as `import {t} from i18next` -- it keeps global state

Reply:
This PR will be closed in favor of #662

Reply:
Now I remember why i did this, because of this error: 
![image](https://github.com/user-attachments/assets/93c6a026-5f0b-4cb0-ba74-4d0a9ad7aa3d)

Reply:
I mean directly `import {t} from i18next` without `useTranslation`